### PR TITLE
Implement liveness/readdiness probe

### PR DESCRIPTION
**Pull request by DmitriyMolch**
https://cerenetwork.atlassian.net/browse/CBI-2170

### CODE REVIEW

File: api/src/services/healthService.js [hunk 1]
```diff
@@ -94,6 +96,33 @@ function splitParams(params) {
...
+async function readiness(req, res) {
+  let errors = [];
+  
+  try {
+    await getClient();
+  } catch (err) {
+    errors.push(`Unable to connect to database:${err.message}`);
+  }
+
+  const accounts = cacheService.getAccounts();
```
Comment (AndreiNavoichyk) [inline api/src/services/healthService.js]: Introduce a new method in `cacheService` with name `isReady` and incapsulate the logic there.

File: api/src/services/healthService.js [hunk 2]
```diff
@@ -94,6 +96,33 @@ function splitParams(params) {
...
+  if(!accounts.length) {
```
Comment (AndreiNavoichyk) [inline api/src/services/healthService.js]: Fix formatting.

File: api/src/services/healthService.js [hunk 3]
```diff
@@ -94,6 +96,33 @@ function splitParams(params) {
...
+    errors.push('Blockchain accounts not cached yet');
+  }
+  
+  errors.length
```
Comment (AndreiNavoichyk) [inline api/src/services/healthService.js:127]: You should also add check that you've connected to all networks.

File: api/src/services/healthService.js (hunk 1 above)
Comment (DmitriyMolch) [reply api/src/services/healthService.js]: fixed

File: api/src/services/healthService.js (hunk 2 above)
Comment (DmitriyMolch) [reply api/src/services/healthService.js]: fixed

File: api/src/services/healthService.js (hunk 3 above)
Comment (DmitriyMolch) [reply api/src/services/healthService.js:127]: fixed

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**